packages/perseus/src/widgets/graded-group-set.jsx

```diff
@@ -51,7 +52,11 @@ class Indicators extends React.Component<IndicatorsProps> {
                             styles.selectedIndicator,
                     )}
                     onClick={() => this.props.onChangeCurrentGroup(i)}
-                />,
+                >
+                    {i === this.props.currentGroup && (
+                        <span className={css(a11y.srOnly)}>current</span>
```

Comment:
Is this going to be read by screenreader?  If so, should we be i18n'ing the string?